examples/blocks.rs

```diff
@@ -31,14 +31,13 @@ async fn main() -> anyhow::Result<()> {
     };
 
     // Subscribe to the block stream with the specified configuration
-    let mut sub = stream.subscribe_with_config(config).await?;
+    let mut sub = stream.subscribe_raw_with_config(config).await?;
 
     println!("Listening for blocks...");
 
     // Process incoming blocks
     while let Some(bytes) = sub.next().await {
-        let message = bytes?;
-        let decoded_msg = Block::decode_raw(message.payload.to_vec()).await;
+        let decoded_msg = Block::decode_raw(bytes).unwrap();
```

Comment:
If we choose to decode the data internally in the SDK, we could get rid of this and avoid one extra step here, improving DX.

Reply:
Yeah, that's the plan. However, we have to make that change progressively. For instance, the benches, load tests, and many other tests need the raw bytes atm. Definitely a post-s3 publishing task.